package.json

```diff
@@ -1027,7 +1027,7 @@
                 {
                     "command": "aws.s3.openFile",
                     "when": "view == aws.explorer && viewItem == awsS3FileNode",
-                    "group": "0@2"
+                    "group": "0@1"
```

Comment:
This is duplicated with the group for `aws.s3.uploadFile` below. Please ensure that these have different values so we have predictable ordering.

Reply:
But `aws.s3.uploadFile` is on view: `view == aws.explorer && viewItem =~ /^(awsS3BucketNode|awsS3FolderNode)$/` meanwhile openFile is on view: `view == aws.explorer && viewItem == awsS3FileNode` so I am guessing there should be no issues?

Reply:
Ack, my bad. Disregard!